src/tweaksConfig.ts

```diff
@@ -0,0 +1,151 @@
+   * nativeHlsParsingEnabled won’t have any effect.
+   *
+   * Default is true.
+   *
+   * @platform iOS
+   */
+  isCustomHlsLoadingEnabled?: boolean;
+  /**
+   * The threshold which will be applied when seeking to the end in seconds. This value will be used
+   * to calculate the maximum seekable time when calling player.seek(time:) orplayer.playlist.seek(source:time:),
+   * so the maximum value will be duration - seekToEndThreshold.
+   *
+   * This is useful if the duration of the segments does not match the duration specified in the
+   * manifest. In this case, if we try to seek to the end, AVPlayer could get stuck and might stall
+   * forever Therefore increasing this value could help.
+   *
+   * Default is 0.5.
+   *
+   * @platform iOS
+   */
+  seekToEndThreshold?: number;
+  /**
+   * Specifies the player behaviour when Player.play is called. Default is 'relaxed'.
+   *
+   * - 'relaxed': Starts playback when enough media data is buffered and continuous playback without stalling can be ensured. If insufficient media data is buffered for playback to start, the player will act as if the buffer became empty during playback.
+   * - 'aggressive': When the buffer is not empty, this setting will cause the player to start playback of available media immediately. If insufficient media data is buffered for playback to start, the player will act as if the buffer became empty during playback.
+   *
+   * @platform iOS
+   */
+  playbackStartBehaviour?: 'relaxed' | 'aggressive';
+  /**
+   * Specifies the player behaviour when stalling should be exited. Default is 'relaxed'.
+   *
+   * - 'relaxed': The player will wait until the buffer is filled that it can, most likely, ensure continuous playback without another stalling right after playback continued.
+   * - 'aggressive': The player will try to unstall as soon as some media data became available and will start playback of this media immediately.
+   *
+   * @platform iOS
+   */
+  unstallingBehaviour?: 'relaxed' | 'aggressive';
+  /**
+   * Constantly aggregated and weighted bandwidth samples are summed up to this weight limit to calculate an bandwidth estimation. Remaining samples (i.e. that would lead to exceeding the limit) are dropped from memory as they are not relevant anymore.
+   * Default is 2000.
+   *
+   * @platform Android
+   */
+  bandwidthEstimateWeightLimit?: number;
+  /**
+   * Some devices have an incorrect implementation of MediaCodec.setOutputSurface. This leads to failure when the surface changes. To prevent failure, the codec will be released and re-instantiated in those scenarios.
+   *
+   * @platform Android
+   */
+  devicesThatRequireSurfaceWorkaround?: {
+    /**
+     * A device name as reported by Build.DEVICE.
+     *
+     * @see Build.DEVICE: https://developer.android.com/reference/kotlin/android/os/Build.html#DEVICE--
+     */
+    deviceNames?: string[]; // Mapped to an array of `DeviceName`s in Kotlin
```

Comment:
```suggestion
    deviceNames?: string[];
```
I am not sure if the comment is of value here 🤔 
Why do you think it is valuable? 🙂

Reply:
Oh, I actually forgot to remove those comments when I finished writing the issue 😅. But I'll remove them, they add no value indeed. Thanks 🙂.

Reply:
Done 59d57708d89c0faf4f5d5aca0a42ea2399153f98